packages/launchpad/src/Main.vue

```diff
@@ -82,14 +92,17 @@ import Spinner from '@cy/components/Spinner.vue'
 import CompareTestingTypes from './setup/CompareTestingTypes.vue'
 import MigrationWizard from './migration/MigrationWizard.vue'
 import ScaffoldedFiles from './setup/ScaffoldedFiles.vue'
-
+import MigrationLandingPage from './migration/MigrationLandingPage.vue'
 import { useI18n } from '@cy/i18n'
 import { computed, ref } from 'vue'
 import LaunchpadHeader from './setup/LaunchpadHeader.vue'
 import OpenBrowser from './setup/OpenBrowser.vue'
+import { useOnline } from '@vueuse/core'
 
 const { t } = useI18n()
 const isTestingTypeModalOpen = ref(false)
+const wasLandingPageShown = ref(false)
```

Comment:
Should this be stored in the backend project state? It is desired that we always show this for every project that needs to be migrated?

Reply:
In so far as this has been described, it's "before migration" so I think this is the best place to start. It will be easy to add conditions in the future if needed, in a follow up PR.